addons/l10n_ro_efactura/models/account_move.py

```diff
@@ -78,15 +86,23 @@ def _l10n_ro_edi_create_document_invoice_sending(self, key_loading, attachment_r
         document.attachment_id = self.env['ir.attachment'].sudo().create(attachment_values)
         return document
 
-    def _l10n_ro_edi_create_document_invoice_sending_failed(self, message, attachment_raw=None):
+    def _l10n_ro_edi_create_document_invoice_sending_failed(self, message, attachment_raw=None, key_loading=None):
+        # TODO in master: use 1 dictionary "values" as the parameter
```

Comment:
Not sure we want to change the signature in master :sweat_smile:

Reply:
it's already changed from 18.0+ 🙈